pkg/common/common.go

```diff
@@ -20,5 +20,7 @@ const (
 	// BackupRestore is the key for the etcd-backup-restore image in image vector
 	BackupRestore = "etcd-backup-restore"
 	// ChartPath is the directory that contains the default image vector file
-	ChartPath = "charts"
+	ChartPath                  = "charts"
+	GardenerOperation          = "gardener.cloud/operation"
+	GardenerOperationReconcile = "reconcile"
```

Comment:
You don't need to introduce these constants. Please use the ones from https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1beta1/constants/types_constants.go instead.

Reply:
I told the same, there is a lot of duplicate code that is maintained in g/g or g/gext already (not only in this PR but also in the already merged code in `master`). @georgekuruvillak explained that he doesn't want to code-wise depend on g/g and g/gext. I'm not sure if this is a good idea, but I suggest to continue with the "copy" approach for now and address this after the PR. WDYT?

Reply:
Hm, I just saw that it already has a dependency to g/g and that's why I thought why not reuse it already.